api-server/source/ddb_patentdata.py

```diff
@@ -0,0 +1,15 @@
+import boto3
+
+# Get the service resource.
+dynamodb = boto3.resource('dynamodb')
+
+# Create the DynamoDB table.
+Patent_data_name = "patent_dynamodb_table"
+Patent_Table = dynamodb.Table(Patent_data_name)
```

Comment:
please use lower case for variables
```suggestion
patent_Table = dynamodb.Table(Patent_data_name)
```

the same is true for Patent_data_name